test/interop/python/multilocale/COMPOPTS

```diff
@@ -1 +1 @@
---library --library-python
+--library --library-python --library-dir=lib
```

Comment:
I'd like to see some tests of multilocale python modules :)

Reply:
I can clone the tests I've added for multilocale? Or just one them.

Reply:
I think cloning the tests would be fine.  I'd do all of them if it isn't too much trouble

Reply:
I've cloned two tests to multilocale.